doc/source/skips/3-transition-to-v1.rst

```diff
@@ -0,0 +1,243 @@
+.. _skip_3_transition_v1:
+
+==========================================
+SKIP 3 — Transitioning to scikit-image 1.0
+==========================================
+
+:Author: Juan Nunez-Iglesias <juan.nunez-iglesias@monash.edu>
+:Status: Draft
+:Type: Standards Track
+:Created: 2021-07-15
+:Resolved:
+:Resolution:
+:Version effective: 0.20
+
+Abstract
+--------
+
+scikit-image is preparing to release version 1.0. This is potentially an
```

Comment:
Some ideas on the structure, @jni. How about inverting the sentences a lil bit? :slightly_smiling_face: 

> scikit-image is preparing to release version 1.0. Despite being in beta and in a 0.x series of releases, scikit-image is used extremely broadly, and any backwards incompatible changes are likely to be disruptive. A new point version presents an opportunity to clean up the API, including backwards incompatible changes. Some of these changes involve changing return values without changing function signatures, which can ordinarily only be done by adding an otherwise useless keyword argument (such as ``new_return_style=True``) whose default value changes over several releases. The result is *still* a backwards incompatible change, but made over a longer time period.
>
> This SKIP proposes a process to ensure that the community is aware of upcoming changes, and can adapt their libraries *or* their declared scikit-image version dependencies accordingly.